### ISSUE TITLE
Linear ODE issue 

### ISSUE THREAD

**Issue by aigerimhatake** [martin-knudsen/masterThesis#2]
When I run the code for linear ODE

inits = torch.tensor(cvqnn_layers_all(L, M, seed=0), dtype=torch.float64) 

Shows an error:  len() of unsized object

How can I fix it?

**Comment by martin-knudsen** [comment]
This code has not been updated since my submission in October 2020. Specifically, I think the cvqnn_layers_all method is obsolete now. I made it work by installing old versions of the packages valid for October 2020:

- Python=8
- pennylane=0.11
- pennylane-sf=0.11
- Sklearn=0.23
- PyTorch=1.6.0
- numpy=1.19.2

 To run with up-to-date versions, one would have to update the syntax.
